saleor/plugins/staff_event/tests/test_notify_events.py

```diff
@@ -0,0 +1,31 @@
+from unittest import mock
+
+from ..tasks import send_staff_create_category_email_task
+
+
+@mock.patch(
+    "saleor.plugins.staff_event.notify_events.send_staff_create_category_email_task."
+    "delay"
+)
+def test_send_staff_create_category_email_task(
+    mocked_email_task, staff_users, staff_event
+):
+    title = "title"
+    content = "content"
+    payload = {
+        "staff_user": staff_users.id,
+        "staff_user_email": staff_users.email,
+        "app": None,
+        "title": title,
+        "content": content,
+        "send_email": True,
+        "domain": "localhost:8000",
+        "site_name": "Saleor",
+    }
+    config = {"host": "localhost", "port": "1025"}
+    send_staff_create_category_email_task(
```

Comment:
nên chia ra thành từng section cho từng mục đích
ví dụ:

Section 1: Prepare variable for test
Section 2: Action
Section 3: Assertion